RELEASE.md

```diff
@@ -1,3 +1,25 @@
+# Release 1.3.0
+
+## New features
+
+- Dockerfile optimisation (https://github.com/marqo-ai/marqo/pull/569/files). A pre-built Marqo base image results in reduced image layers and increased build speed.
+- New E5 models added to model registry (https://github.com/marqo-ai/marqo/pull/568). E5 V2 and Multilingual E5 models are now available for use. See all available models [here] (https://marqo.pages.dev/1.2.0/Models-Reference/dense_retrieval/).
+
+## Bug fixes and minor changes
+
+- Improved testing pipeline (https://github.com/marqo-ai/marqo/pull/582 & https://github.com/marqo-ai/marqo/pull/586). Tests now trigger on pull request updates. This results in safer and easier merges to mainline.
+- Major README overhaul (https://github.com/marqo-ai/marqo/pull/573). The README has been revamped with up-to-date examples and easier to follow instructions.
+- New security policy (https://github.com/marqo-ai/marqo/pull/574).
+- Updated requirements file. Now the `requirements.dev.txt` should be used to install requirements for development environments (https://github.com/marqo-ai/marqo/pull/569). Pins for `protobuf` & `onnx` have been removed while a pin for `anyio` has been added (https://github.com/marqo-ai/marqo/pull/581, & https://github.com/marqo-ai/marqo/pull/589).
```

Comment:
Let's reduce ambiguity for unfamiliar readers: `Pins for...` -> `Version pins for...`

Reply:
nit: `Updated requirements file.` -> `Updated requirements files.`